modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,65 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline, resource level
+
+By default, policies are applied to the entire API. However, you can implement an additional level of granularity for policies, where access is controlled based on the a criteria, called resource-level policies. 
+
+In this level of granularity, policies are applied to only those request that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+//* SLA-based policies
+//Impose a limit specified by the level of access granted to the requesting application.The Rate Limiting policy is SLA-based.
+// Is SLA based a policy level granularity or a configuration level? because it doesn't change where the policy is applied, just the configuration ( a.k.a limit) - Eva??
+
```

Comment:
FYI, I should not be editing a file that has questions outstanding to your SMEs. I am supposed to be editing content that has been technically reviewed and resolved.

Reply:
noted.